script/formatting/formatter.py

```diff
@@ -0,0 +1,233 @@
+            os.system(autopep_command)
+
+            # AUTOFLAKE
+            autoflake_command = AUTOFLAKE_BINARY + \
+                " --in-place --remove-all-unused-imports"\
+                " --remove-unused-variables " + file_path
+            #LOG.info(autoflake_command)
+            os.system(autoflake_command)
+
+    # END WITH
+
+    fd.close()
+# END FORMAT__FILE(FILE_NAME)
+
+
+# format all the files in the dir passed as argument
+def format_dir(dir_path, add_header, strip_header, format_code):
+
+    for subdir, dir, files in os.walk(dir_path):
+        for file in files:
+            # print os.path.join(subdir, file)
+            file_path = subdir + os.path.sep + file
+
+            if file_path.endswith(".py"):
+                format_file(file_path, add_header, strip_header, format_code)
+            # END IF
+        # END FOR [file]
+    # END FOR [os.walk]
+# END ADD_HEADERS_DIR(DIR_PATH)
+
+# ==============================================
+# Main Function
+# ==============================================
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description='Add/delete headers and/or format source code')
+
+    parser.add_argument("-a", "--add-header", help='add suitable header(s)',
+                        action='store_true', default=False)
+    parser.add_argument("-s", "--strip-header", help='strip existing header(s)',
+                        action='store_true', default=False)
+    parser.add_argument("-c", "--format-code", help='format code',
+                        action='store_true', default=True)
+    parser.add_argument("-f", "--file-name",
+                        help='file to be acted on')
+    parser.add_argument("-d", "--dir-name",
+                        help='directory containing files to be acted on')
+
+    args = parser.parse_args()
+
+    # SOME SANITY CHECKS
+    if args.add_header and args.strip_header:
+        LOG.error("adding & stripping headers cannot be done together")
+        sys.exit("adding & stripping headers cannot be done together")
+    if args.file_name and args.dir_name:
+        LOG.error("file_name and dir_name cannot be specified together")
+        sys.exit("file_name and dir_name cannot be specified together")
+
+    if args.file_name:
+        LOG.info("Scanning file: " + ''.join(args.file_name))
+        format_file(args.file_name, args.add_header,
+                    args.strip_header,
+                    args.format_code)
+    elif args.dir_name:
+        LOG.info("Scanning directory " + ''.join(args.dir_name))
+        format_dir(args.dir_name,
+                   args.add_header,
+                   args.strip_header,
+                   args.format_code)
+    # BY DEFAULT, WE SCAN THE DEFAULT DIRS AND FIX THEM
+    else:
+        LOG.info("Default scan")
+        for dir in DEFAULT_DIRS:
+            LOG.info("Scanning : " + dir + "\n\n")
+
+#             LOG.info("Stripping headers : " + dir)
```

Comment:
Any reason for having this commented code? 
Can we remove this?

Reply:
Uncommented the code.